docs/_includes/default.html

```diff
@@ -13,7 +13,7 @@
     <link rel="stylesheet" href="css/normalize.css" />
     <link rel="stylesheet" href="css/style.css" />
     <link rel="stylesheet" href="css/prism.css" />
-    <link rel="shortcut icon" href="favicon.ico" />
+    <link rel="icon" href="favicon.ico" />
```

Comment:
BTW you can remove the `/>` from the HTML tags here. It's not needed in HTML5+.

Reply:
Forgot to mention, `shortcut icon` is needed for old IE. Which AFAICT since html5shiv is loaded, it is still supported.

If not, that's OK, but both should be dropped.